tools/DfmHttpService/Model/CommandMessage.cs

```diff
@@ -19,16 +19,19 @@ internal class CommandMessage
         [JsonProperty("markdownContent")]
         public string MarkdownContent { get; set; }
 
-        [JsonProperty("writeTempPreviewFile")]
-        public bool WriteTempPreviewFile { get; set; }
+        [JsonProperty("shouldWriteTempPreviewFile")]
+        public bool ShouldWriteTempPreviewFile { get; set; }
 
-        [JsonProperty("previewFilePath")]
-        public string PreviewFilePath { get; set; }
+        [JsonProperty("shouldSeparateMarkupResult")]
+        public bool ShouldSeparateMarkupResult { get; set; }
+
+        [JsonProperty("tempPreviewFilePath")]
+        public string TempPreviewFilePath { get; set; }
 
         [JsonProperty("pageRefreshJsFilePath")]
         public string PageRefreshJsFilePath { get; set; }
```

Comment:
add TODO. implement it after DocfxPreview

---
In reply to: [113358892](https://github.com/dotnet/docfx/pull/1590#discussion_r113358892) [](ancestors = 113358892)